File: ProjekatApi/ProjekatApi/FormModel/BusinessReport.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ProjekatApi.FormModel
{
    public class BusinessReport
    {
        public DateTime Today { get; set; }

        public string ratingPerDay { get; set; }

        public string ratingPerWeek { get; set; }

        public string ratingPerMonth { get; set; }
    }
}

File: Projekat/src/app/entities/businessReport/business-report.ts
import { DatePipe } from '@angular/common';

export class BusinessReport {
    Today: DatePipe;

    ratingPerDay: string;

    ratingPerWeek: string;

    ratingPerMonth: string;

    constructor(today: DatePipe, ratingPerDay: string, ratingPerWeek: string, ratingPerMonth: string){
        this.Today = today;
        this.ratingPerDay = ratingPerDay;
        this.ratingPerMonth = ratingPerMonth;
        this.ratingPerWeek = ratingPerWeek;
    }
}
